### PR TITLE
Fix typos in the docs

### PR DESCRIPTION
**Pull request by TheophileDiot**

### CODE REVIEW

File: docs/integrations.md [hunk 1]
```diff
@@ -330,11 +330,11 @@ networks:
 </figure>
 
 !!! info "Docker autoconf"
-    The Docker autoconf integration is similar of the Docker autoconf one (but with services instead of containers). Please read the [Docker autoconf integration section](#docker-autoconf) first if needed.
+    The Docker autoconf integration is similar to the Docker autoconf one (but with services instead of containers). Please read the [Docker autoconf integration section](#docker-autoconf) first if needed.
```
Comment (fl0ppy-d1sk) [inline docs/integrations.md:333]: The Docker swarm integration is similar to the Docker autoconf one ...

File: docs/integrations.md [hunk 2]
```diff
@@ -849,7 +849,7 @@ Repositories of Linux packages for BunkerWeb are available on [PackageCloud](htt
 	chown -R root:nginx /opt/bunkerweb
 	```
 	
-	Last but not least, you will need to setup systemd unit files :
+	Last but not least, you will need to set up system unit files :
```
Comment (fl0ppy-d1sk) [inline docs/integrations.md:852]: set up systemd

File: docs/migrating.md [hunk 3]
```diff
@@ -21,16 +21,16 @@ Instead of supporting only Authelia, we decided to support generic auth request
 
 ## Replaced BLOCK_\*, WHITELIST_\* and BLACKLIST_\* settings
 
-The blocking mechanisms has been completely redesigned. We have detected that a lot of false positives came from the default blacklists hardcoded into bunkerized. That's why we decided to give the users the choice of their blacklists (and also whitelists) for IP address, reverse DNS, user-agent, URI and ASN, see the [Blacklisting and whitelisting](/1.4/security-tuning/#blacklisting-and-whitelisting) section of the [security tuning](/1.4/security-tuning).
+The blocking mechanisms have been completely redesigned. We have detected that a lot of false positives came from the default blacklists hardcoded into bunkerized. That's why we now give users the possibility of choosing their own blacklists (and also whitelists). For IP address, reverse DNS, user-agent, URI and ASN, see the [Blacklisting and whitelisting](/1.4/security-tuning/#blacklisting-and-whitelisting) section of the [security tuning](/1.4/security-tuning).
```
Comment (fl0ppy-d1sk) [inline docs/migrating.md:24]: Single sentence : ... (and also whitelists) for IP address ...

File: docs/plugins.md [hunk 4]
```diff
@@ -412,12 +412,12 @@ Content of foo is : {{ foo }}.
...
 ```
 
-Please note that every form submission is protected via a CSRF token, you will need to include the follow snippet into your forms :
+Please note that every form submission is protected via a CSRF token, you will need to include the following snippet into your forms :
 ```html
 <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
 ```
 
-Retrieving user submitted data is pretty simple thanks to the request module provided by Flask :
+Retrieving user submitted data is pretty simple, thank to the request module provided by Flask :
```
Comment (fl0ppy-d1sk) [inline docs/plugins.md:420]: waiting confirmation but I thought "thanks to" is the right one

File: docs/quickstart-guide.md [hunk 5]
```diff
@@ -275,7 +275,7 @@ You will find more settings about reverse proxy in the [settings section](/1.4/s
 
 === "Ansible"
 
-    We will assume that you already have a service running and you want to use bunkerweb as a reverse-proxy.
+    We will assume that you already have a service running and you want to use Bunkerweb as a reverse-proxy.
```
Comment (fl0ppy-d1sk) [inline docs/quickstart-guide.md:278]: BunkerWeb (W caps)

File: docs/quickstart-guide.md [hunk 6]
```diff
@@ -1048,7 +1048,7 @@ REAL_IP_HEADER=X-Forwarded-For
 
 === "Kubernetes"
 
-    You will need to add the settings to the environment variables of the bunkerweb containers (doing it using the ingress is not supported because you will get into trouble when using things like Let's Encrypt) :
+    You will need to add the settings to the environment variables of the Bunkerweb containers (doing it using the ingress is not supported because you will get into trouble when using things like Let's Encrypt) :
```
Comment (fl0ppy-d1sk) [inline docs/quickstart-guide.md:1051]: BunkerWeb (W caps)

File: docs/quickstart-guide.md [hunk 7]
```diff
@@ -1085,7 +1085,7 @@ REAL_IP_HEADER=X-Forwarded-For
 	...
 	```
 
-    Don't forget to restart the bunkerweb service once it's done.
+    Don't forget to restart the Bunkerweb service once it's done.
```
Comment (fl0ppy-d1sk) [inline docs/quickstart-guide.md:1088]: BunkerWeb (W caps)

File: docs/quickstart-guide.md [hunk 8]
```diff
@@ -1218,7 +1218,7 @@ REAL_IP_HEADER=proxy_protocol
 
 === "Kubernetes"
 
-    You will need to add the settings to the environment variables of the bunkerweb containers (doing it using the ingress is not supported because you will get into trouble when using things like Let's Encrypt) :
+    You will need to add the settings to the environment variables of the Bunkerweb containers (doing it using the ingress is not supported because you will get into trouble when using things like Let's Encrypt) :
```
Comment (fl0ppy-d1sk) [inline docs/quickstart-guide.md:1221]: BunkerWeb (W caps)

File: docs/quickstart-guide.md [hunk 9]
```diff
@@ -1258,7 +1258,7 @@ REAL_IP_HEADER=proxy_protocol
 	...
 	```
 
-    Don't forget to restart the bunkerweb service once it's done.
+    Don't forget to restart the Bunkerweb service once it's done.
```
Comment (fl0ppy-d1sk) [inline docs/quickstart-guide.md:1261]: BunkerWeb (W caps)

File: docs/quickstart-guide.md [hunk 10]
```diff
@@ -1520,7 +1520,7 @@ Some integrations offer a more convenient way of applying configurations for exa
     chmod -R 770 /opt/bunkerweb/configs
     ```
 
-    Don't forget to restart the bunkerweb service once it's done.
+    Don't forget to restart the Bunkerweb service once it's done.
```
Comment (fl0ppy-d1sk) [inline docs/quickstart-guide.md:1523]: BunkerWeb (W caps)

File: docs/plugins.md (hunk 4 above)
Comment (TheophileDiot) [reply docs/plugins.md:420]: Both of them can be used, which one do you prefer ?

**Review by fl0ppy-d1sk — APPROVED** [review]

**Pull request merged**